### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by tajulafreen**
In this PR, I did the following  :
1. Implemented the desktop version and made it look like the one in the Figma design template that I choose.
2. Use media queries in my CSS file to make my desktop version look like the template I selected.
3. I used the correct GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,15 +17,26 @@
...
+    <span><i class="fa fa-bars mobile"></i></span>
+
+    <!--------------- Desktop ------------------->
+    <ul class="navbar-link desktop">
+     <li><a href="#flexcontainer" class="navbar-item">Portfolio</a></li>
+     <li><a href="#about-me" class="navbar-item">About Me</a></li>
+     <li><a href="#form-contact" class="navbar-item">Contact</a></li>
+    </ul>
+
+        <!--------------- Desktop ------------------->
```
Comment (usorfaitheloho) [inline index.html:33]: According to the project requirement, you are to ensure that your design is the same as the Figma design given. Your desktop version still has the `hamburger` menu which is meant for mobile only. Please make the `hamburger` invisible for the desktop version only and then make this `desktop nav` you added visible for the desktop only, and kindly make the desktop nav centered 😄 

Your design : 
image


Figma design :
image

File: index.html [hunk 2]
```diff
@@ -16,15 +17,26 @@
...
  </nav>
  </header>
  
 <section>
-<div class="flexcontainer">
+<div class="flexcontainer section" id="flexcontainer">
+    
     <div class="flexitem heading"><h1>Hey there. I’m Afreen.<br><b>I’m a software developer.</b></h1></div>
     <div class="flexitem paragraph"><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p></div>
```
Comment (usorfaitheloho) [inline index.html:42]: As stated earlier, please you're to follow the design in the Figma template you were given both for your background images and for your text alignment too. All your background images are supposed to be at the edge surrounding your header in a proper and orderly manner while your header is in the center as shown in the Figma design then your social icons should be arranged too. please kindly implement that for your design 😄 

Figma design :
image



Your design ;
image

File: responsiveness.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+  .grid-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    width: 45%;
+    margin-left: 12%;
+    margin-bottom: 15%;
+    gap: 3%;
+  }
+
```
Comment (usorfaitheloho) [inline responsiveness.css:94]: According to the project requirement, you are to ensure that your design is the same as the Figma design given. Please Your Project section for the desktop version should have  2 rows with 3 projects each (6 projects in total) to give you a better view of what we have in the template also remember to follow all alignments for this section 😄 .

Figma design:

https://github.com/tajulafreen/Portfolio/assets/20537712/35b83063-4487-42fb-a667-cbd68caf92ff



Your design:

https://github.com/tajulafreen/Portfolio/assets/20537712/935a843a-fbb6-4383-a012-7fd63218e594

File: responsiveness.css [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  .footer {
+    margin-top: 25%;
+    margin-right: 15%;
+    margin-bottom: 3%;
+  }
+
+  .skills {
+    margin-left: 35%;
+  }
+}
```
Comment (usorfaitheloho) [inline responsiveness.css:229]: Please as I've suggested previously that you should follow the Figma design for the desktop version as much as possible, please kindly implement it for other aspects of your desktop version which also includes: the `About section` and `Contact section` . part of the problem you have with your code could be that you have several css files with conflicting class properties. You could try to have just one or two maximum css files and ensure that the properties are unique for each class 😄 

Figma design : 

https://github.com/tajulafreen/Portfolio/assets/20537712/2ab407ef-cbdc-4e22-928a-03b1011adecd

Your design :


https://github.com/tajulafreen/Portfolio/assets/20537712/b924d154-9e73-4304-a5c5-2cd3453d67c1

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @tajulafreen   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Git/Github flow best practices has been followed :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 
:arrow_forward: Good PR title and summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @tajulafreen, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**